src/dotnet/commands/dotnet-new/Program.cs

```diff
@@ -26,6 +26,13 @@ private static string GetFileNameFromResourceName(string s)
 
         public int CreateEmptyProject(string languageName, string templateDir)
         {
+            // Check if project.json exists in the folder
+            if (File.Exists(Path.Combine(Directory.GetCurrentDirectory(), "project.json")))
```

Comment:
There is already a constant for it:  https://github.com/dotnet/cli/blob/rel/1.0.0-preview2/src/Microsoft.DotNet.ProjectModel/Project.cs#L16